qiskit_ibm_runtime/ibm_runtime_service.py

```diff
@@ -129,54 +117,68 @@ class IBMRuntimeService:
 
     def __init__(
         self,
-        auth: Optional[Literal["cloud", "legacy"]] = None,
         token: Optional[str] = None,
-        locator: Optional[str] = None,
-        **kwargs: Any,
+        url: Optional[str] = None,
+        instance: Optional[str] = None,
+        auth: Optional[AccountType] = None,
+        name: Optional[str] = None,
+        proxies: Optional[dict] = None,
+        verify: Optional[bool] = None,
     ) -> None:
         """IBMRuntimeService constructor
 
         Args:
-            auth: Authentication type. `cloud` or `legacy`. If not specified, the saved default is used.
-                If there is no default value, and both accounts were saved on disk, the cloud type is
-                used.
-            token: Token used for authentication. If not specified, the saved token is used.
-            locator: The authentication url, if `auth=legacy`. Otherwise the CRN.
-            **kwargs: Additional settings for the connection:
-
-                * proxies (dict): proxy configuration.
-                * verify (bool): verify the server's TLS certificate.
+            token: IBM Cloud API key or IBM Quantum API token.
+            url: The API URL.
+                Defaults to https://cloud.ibm.com (cloud) or https://auth.quantum-computing.ibm.com/api (legacy).
+            instance: The CRN (cloud) or hub/group/project (legacy).
+            auth: Authentication type. `cloud` or `legacy`.
+            name: Name of the account to load.
+            proxies: Proxy configuration for the server.
+            verify: If False, ignores SSL certificates errors
```

Comment:
`verify` doesn't really ignore errors; it just doesn't verify the server's certificate (like what the old comment said).

Reply:
Updated comment (https://github.com/Qiskit/qiskit-ibm-runtime/pull/49/commits/9fd5a600c08e2b90a5222157d27bdf08e1f834a7)